src/api/spec/policies/project_policy_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe ProjectPolicy do
+  let(:anonymous_user) { create(:user_nobody) }
+  let(:user) { create(:confirmed_user) }
+  let(:other_user) { create(:confirmed_user) }
+  let(:admin_user) { create(:admin_user) }
+  let(:project) { user.home_project }
+  subject { ProjectPolicy }
+
+  permissions :create?, :can_create_package_in?, :unlock?, :local_project_and_allowed_to_create_package_in? do
+    it { expect(subject).to permit(user, project) }
+    it { expect(subject).not_to permit(anonymous_user, project) }
+    it { expect(subject).not_to permit(other_user, project) }
+    it { expect(subject).to permit(admin_user, project) }
+  end
+
+  permissions :local_project_and_allowed_to_create_package_in? do
```

Comment:
This is a private method in the policy, no need to test it

Reply:
I was probably not private when @vpereira wrote the test, as this was introduced in https://github.com/openSUSE/open-build-service/commit/3e226956eb594acaa8cf875bd61dbafa8498c0da

It makes now no sense to test it.

Reply:
it wasn't private, and ```show?``` and ```index?``` didn't exist @mdeniz